src/index.ts

```diff
@@ -10,5 +11,8 @@ const app = express();
 server.applyMiddleware({ app });
 app.use('/', express.static('public'));
 
-const port = process.env.SERVER_PORT || 4000;
-app.listen(port, () => console.log(`App listening on port ${port}!`));
+const port = config.server!.port;
```

Comment:
Here you can add default port for a start like it was before `const port = config.server!.port || 4444;`
When you will remove `port` and try to run API you will get an undefined port for start

Reply:
@kanoru3101 any updates here?